src/clusterfuzz/_internal/base/data_import.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Data retrieving helper functions."""
+
+import ast
+import base64
+
+from clusterfuzz._internal.base import utils
+from clusterfuzz._internal.google_cloud_utils import storage
+
+
+def get_url_content(url):
```

Comment:
Please move these functions back into revisions, as it's very specific to that.

Other callers can call `storage.read_data` directly.

Reply:
as i read through the function, it seemed to work really for us to try and import the `.json` config we added. It seems to designed for the `.json` format more generally. Therefore, to avoid repeating a lot of this code in `android.settings`, I thought it'd be better to make it available outside of `revisions.py`. But I think I see your point, we might want to change the way configs are written, in which case, we'd have to stop relying on this function anyway. In that case, it'd just be left hanging outside. Do I understand it correctly?